mlrun/db/httpdb.py

```diff
@@ -3143,6 +3143,21 @@ def verify_authorization(
             body=dict_to_json(authorization_verification_input.dict()),
         )
 
+    def list_api_gateways(self, project_name=None):
+        """
+        Returns the list of Nuclio api gateways
+        :param project_name: optional str parameter to filter by project, if not passed, default Nuclio's value is taken
+
+        :return: json with the list of Nuclio Api Gateways
+            (json example is here https://github.com/nuclio/nuclio/tree/development/cmd/dashboard#response-20)
+        """
+        params = {}
+        if project_name:
+            params["project-name"] = project_name
+        error = "list api gateways"
+        resp = self.api_call("GET", "api-gateways", error, params)
```

Comment:
shouldn't the url be `/projects/{project}/nuclio/api-gateways`?
How does it know to go to `nuclio`?

Reply:
@TomerShor nice catch! That is indeed true